apex_rostest/apex_launchtest/apex_launchtest/apex_runner.py

```diff
@@ -229,6 +229,12 @@ def run(self):
             try:
                 worker = _RunnerWorker(run, self._launch_file_arguments, self._debug)
                 results[run] = worker.run()
+            except unittest.case.SkipTest as skip_exception:
+                # If a 'skip' decorator was placed on the generate_launch_description function,
```

Comment:
@pbaughman leftover
```suggestion
                # If a 'skip' decorator was placed on the generate_test_description function,
```
?